strawberryfields/backends/tfbackend/ops.py

```diff
@@ -70,7 +70,7 @@ def _numer_safe_power(base, exponent, dtype=tf.complex64):
     return base ** exponent
 
 
-def mixed(pure_state, batched=False):
+def mix(pure_state, batched=False):
     """Converts the state from pure to mixed"""
     # todo: In the fock backend mixing is done by ops.mix(), maybe the functions should be named identically?
```

Comment:
Looks like you can delete this TODO comment now, @Aaron-Robertson :)

Reply:
> Oh, that's awesome. Is this generally the case that broadcasting is faster than tensordot? Or just for this particular application?

Here we used it for a batched/nonbatched outer product, but it seems to be the case for a few applications, yes:
```python
T1 = tf.constant(np.random.normal(size=(13,5,11,17)))
T2 = tf.constant(np.random.normal(size=(13,5,11,17)))
```
Tensordot runtime:
```python
%timeit tf.tensordot(T1,T2,[[0,1,2,3],[0,1,2,3]])
# 127 µs ± 950 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```
broadcasting is about 5x faster
```python
%timeit tf.reduce_sum(T1*T2)
# 25.2 µs ± 989 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```
but `einsum` seems even better:
```python
%timeit tf.einsum('ijkl,ijkl', T1, T2)
# 20.9 µs ± 791 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```
The only advantage at this point is that the broadcasting approach may make the code simpler with respect to `einsum` (no convoluted strings of indices to generate separately).